spec/app/domain/authentication/authn_ldap/authenticator_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Authentication::AuthnLdap::Authenticator do
+  let(:authenticator_instance) do
+    Authentication::AuthnLdap::Authenticator.new(env: {})
+  end
+  
+  let(:input) do
+    ::Authentication::Strategy::Input.new(
+      authenticator_name: 'ldap',
+      service_id: 'test',
+      account: 'test',
+      username: username,
+      password: password,
+      origin: '127.0.0.1',
+      request: nil
+    )
+  end
+
+  before do
+    allow_any_instance_of(Net::LDAP)
+      .to receive(:bind_as)
+      .and_return(:valid_login?)
```

Comment:
should this be `and_return(true)` or `and_return(<some obj>)`?

Reply:
Good catch! I had intended this to make it easier to control the mocked LDAP response, but I ended up only needing `true` for these cases. Thanks!